src/useFirebaseUser.js

```diff
@@ -40,10 +40,23 @@ const setAuthCookie = async (firebaseUser) => {
       )
     }
   }
-
   return response
 }
 
+const setAuthCookie = async (firebaseUser) => {
+  const { tokenChangedHandler } = getConfig()
+  const authUser = createAuthUser({
+    firebaseUserClientSDK: firebaseUser,
+    clientInitialized: true,
```

Comment:
@kmjennison can you verify this is as expected? Since this is a callback from `firebase.auth` I presume this value should be true.

Reply:
Yes, that's correct. `clientInitialized` basically a flag for whether the client-side JS SDK has loaded, which is guaranteed here.